graphql_compiler/tests/test_data_tools/schema.sql

```diff
@@ -27,7 +27,7 @@ CREATE INDEX Entity_Related ON Entity_Related (in, out) UNIQUE_HASH_INDEX
 ### Event ###
 CREATE CLASS Event EXTENDS Entity
 
-CREATE PROPERTY Event.event_date Date
+CREATE PROPERTY Event.event_date DateTime
```

Comment:
Remember that we have data for snapshot tests that gets loaded into the database, and will need to be updated to load `DateTime` data instead of `Date` data. This data is generated by scripts, and those scripts might need updating as well.

Reply:
The snapshot tests seem to already be using datetimes instead of date. 
('fed_at_datetimes_list', ('2008-09-23T00:00:00Z')),
('child_birthdays_list', ('2012-11-02'))